Games/types/Mafia/roles/cards/MakeKillStronger.js

```diff
@@ -14,18 +14,18 @@ module.exports = class MakeKillStronger extends Card {
           return !this.data.hasUsedStrength;
         },
         action: {
-          labels: ["absolute"],
+          labels: ["strong"],
```

Comment:
there should be an accompanying use of the "strong" label

Reply:
added